### DISCUSSION TITLE
q-menu & q-btn-dropdown propagation issue on q-table row

### DISCUSSION THREAD

**Discussion by rizkybasnur** [quasarframework/quasar#13005] [General - Components / Directives / etc]
Hi! i have a q-table that when we click the row area we can open a dialog based on it rows data. But i also have a q-menu and q-btn-dropdown inside the row. when i click either q-menu or q-btn-dropdown, i notice that the menu and the dropdown is open, but also the dialog. how can i prevent the dialog to be opened when i click q-menu or q-btn-dropdown? is there a props for stopping propagation for q-menu and q-btn-dropdown?

here's the sample of how i put the q-menu and q-btn dropdown inside the q-td tag from q-table. Also i put the activator for the row modal inside the q-td of another section of the row.

`<q-td key="action" class="text-center q-gutter-xs" :props="props">
                <q-btn
                  label="Detail"
                  unelevated
                  class="btn-secondary-2"
                  padding="4px 8px"
                  no-caps
                  @click.stop="setDetail(props.row)"
                />

                <q-btn
                  label="Batalkan"
                  unelevated
                  class="btn-secondary-2"
                  padding="4px 8px"
                  no-caps
                  icon-right="mdi-chevron-down"
                >
                  <q-menu fit :offset="[-10, 10]">
                    <q-list>
                      <q-item clickable v-close-popup @click="selectWarehouse(props.row)">
                        <q-item-section>
                          <q-item-label>Kirim dari gudang sendiri</q-item-label>
                        </q-item-section>
                      </q-item>

                      <q-item clickable v-close-popup @click="cancellation(props.row, 0, {})">
                        <q-item-section>
                          <q-item-label>Batalkan Transaksi</q-item-label>
                        </q-item-section>
                      </q-item>
                    </q-list>
                  </q-menu>
                  </q-btn> 

                <q-btn-dropdown
                  class="btn-secondary-2"
                  padding="4px 8px"
                  label="Batalkan"
                  no-caps
                  unelevated
                  v-if="props.row.isCancelable"
                >
                  <q-list>
                    <q-item clickable v-close-popup @click="selectWarehouse(props.row)">
                      <q-item-section>
                        <q-item-label>Kirim dari gudang sendiri</q-item-label>
                      </q-item-section>
                    </q-item>

                    <q-item clickable v-close-popup @click="cancellation(props.row, 0, {})">
                      <q-item-section>
                        <q-item-label>Batalkan Transaksi</q-item-label>
                      </q-item-section>
                    </q-item>
                  </q-list>
                </q-btn-dropdown>
              </q-td>`

**Comment by RoelRoel** [comment]
Have you tried adding `@click.stop` attribute without value to q-btn-dropdown?
